store/listings/management/commands/fetch_listings.py

```diff
@@ -37,6 +40,17 @@ def handle(self, *args, **options):
                 'filename': 'cotton_eur.txt'
             },
         ]
-
         for listings in listings_data:
-            save_to_file(listings=listings)
+            for listing in listings['data']:
+                listing_object = Listing.objects.create(
+                    title=listing.get('title'),
+                    description=listing.get('description'),
+                    price=listing.get('price'),
+                    currency=listing.get('currency'),
+                    category=listings['category']
+                )
+
+                listing_object.uid = listing.get('listing_id')
```

Comment:
The current implementation does not account for the fact that if a listing with such `uid` already exists in DB.

This code will crash.

If listing with such `uid` exists in db, listing object creation should be skipped without exceptions.